cmd/http-tracer.go

```diff
@@ -80,6 +81,17 @@ func (r *recordRequest) Data() []byte {
 	return logger.BodyPlaceHolder
 }
 
+var ldapPwdRegex = regexp.MustCompile("(^.*?)LDAPPassword=([^&]*?)(&(.*?))?$")
+
+// redact LDAP password if part of string
+func redactLDAPPwd(s string) string {
+	if ldapPwdRegex.MatchString(s) {
+		parts := ldapPwdRegex.FindStringSubmatch(s)
+		return parts[1] + "LDAPPassword=AAAAAAA" + parts[3]
```

Comment:
Will we get sued if someone already uses 'AAAAAAA' as a password ?

Reply:
With that logic any character will be treated that way @vadmeste :P

Reply:
> With that logic any character will be treated that way @vadmeste :P

What about 'StarTrek' ?